docs/content/preview/api/ycql/dml_transaction.md

```diff
@@ -32,13 +32,14 @@ transaction_block ::= BEGIN TRANSACTION
                       END TRANSACTION ';'
 ```
 
-Where
+Where `insert`, `update`, and `delete` are [INSERT](../dml_insert), [UPDATE](../dml_update/), and [DELETE](../dml_delete/) statements.
 
-- `insert`, `update`, and `delete` are [INSERT](../dml_insert), [UPDATE](../dml_update/), and [DELETE](../dml_delete/) statements.
+- When using `BEGIN TRANSACTION`, you don't use a semicolon.
```

Comment:
May be one line for syntax with 'BEGIN TRANSACTION' and another line for syntax with 'START TRANSACTION'? Like

L 37 --> When using `BEGIN TRANSACTION`, you don't use a semicolon. End the transaction block with `END TRANSACTION ;`

L38 --> When using `START TRANSACTION ;`, End the transaction block with `COMMIT ;`